src/fe-gtk/chanview-tree.c

```diff
@@ -394,3 +393,14 @@ cv_tree_is_collapsed (chan *ch)
 	
 	return ret;
 }
+
+static void
+cv_tree_queue_draw (chanview *cv)
+{
+	/* assign the model again to cause the tree view to be redrawn
+	   note: Simply using gtk_widget_queue_draw() is not sufficient. */
```

Comment:
This seems like a hacky workaround, I'd prefer figuring out a way to just tell it to redraw.

Reply:
I've looked at the documentation and tried several things before coming up with that hack. Do you have a specific API function in mind? I only found that function I mention in the comment but it is not sufficient because the layout is still not updated.

The preferred way for this to work is likely that GTK triggers the layout update itself if relevant properties like the font change. So doing things manually here will always be not the optimal solution and therefore a "hacky workaround".